sda/cmd/ingest/ingest_test.go

```diff
@@ -723,3 +735,9 @@ func (ts *TestSuite) TestIngestFile_missingFile() {
 
 	assert.Equal(ts.T(), "ack", ts.ingest.ingestFile(corrID, message))
 }
+func (ts *TestSuite) TestDetectMisingC4GHKeys() {
+	viper.Set("c4gh.privateKeys", "")
+	privateKeys, err := config.GetC4GHprivateKeys()
+	assert.NoError(ts.T(), err)
+	assert.Equal(ts.T(), 0, len(privateKeys))
+}
```

Comment:
What does this part check? Does it check that `config.GetC4GHprivateKeys` translates an empty string to an empty array?

Reply:
Sanity check for the current functionality.
It should be possible to have the function to return an error if the marshaled struct is empty. It was a bit to time consuming to get it to work for this PR.